client/src/models/results.js

```diff
@@ -190,7 +190,7 @@ class Indicator {
   get target_date(){
     const { target_month, target_year } = this;
     if( _.isNumber(target_month) && _.isNumber(target_year) ){
-      return `${months[target_month].text} ${target_year}`;
+      return new Date(target_year, target_month);
```

Comment:
This whole date thing's rough, it makes the `_.isNumber(target_month) && _.isNumber(target_year)` case for `target_date` values a weird special case that every consumer of the Indicator target dates in the InfoBase will now have to write special case code to handle. 

`result_flat_table.js` has access to the raw indicator month and year values (try `_.pick(indicator, ['target_month', 'target_year'])`), can leave this code as it was and keep the special case requiring `Date` objects more contained.

Reply:
Okay, that was my initial direction as well but I decided to change the source `get` as `target_date` was only being used at 2 places. I'll contain this date thing within this.